addons/project/models/project.py

```diff
@@ -1516,7 +1522,7 @@ def _compute_recurrence_message(self):
             else:
                 recurrence_title = _('A new task will be created on the following dates:')
             task.recurrence_message = '<p><span class="fa fa-check-circle"></span> %s</p><ul>' % recurrence_title
-            task.recurrence_message += ''.join(['<li>%s</li>' % date.strftime(date_format) for date in recurring_dates[:5]])
+            task.recurrence_message += ''.join(['<li>%s %s</li>' % (format_date(date, "EEE", locale=self.env.user.lang), date.strftime(date_format)) for date in recurring_dates[:5]])
```

Comment:
```suggestion
            task.recurrence_message += ''.join(['<li>%s %s</li>' % (
                format_date(date, "EEE", locale=self.env.user.lang), date.strftime(date_format)
            ) for date in recurring_dates[:5]])
```
...to see it a little clearer